documentation/src/main/asciidoc/reference/search-dsl-sort.asciidoc

```diff
@@ -376,6 +376,10 @@ The behavior for missing values can be controlled explicitly through the `.missi
 
 * `.missing().first()` puts documents with no value in first position (regardless of the sort order).
 * `.missing().last()` puts documents with no value in last position (regardless of the sort order).
+* `.missing().lowest()` depends on the sort order. In the case of ascending order, the documents with no value will go first.
+The documents with no value will go last if descending order is used.
+* `.missing().highest()` depends on the sort order. In the case of ascending order, the documents with no value will go last.
+The documents with no value will go first if descending order is used.
```

Comment:
Maybe this would be more straightforward?

```suggestion
* `.missing().lowest()` interprets missing values as the lowest value: its puts documents with no value in first position when using ascending order, or in last position when using descending order.
* `.missing().highest()` interprets missing values as the highest value: its puts documents with no value in last position when using ascending order, or in first position when using descending order.
```